<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateAsDialog</class>
 <widget class="QDialog" name="GenerateAsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fileExtension">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.bmp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.jpeg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.png</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.ppm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.xbm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.xpm</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
